sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -126,10 +136,6 @@ public static OpenTelemetryBuilder UseAzureMonitor(this OpenTelemetryBuilder bui
             {
                 logging.AddOpenTelemetry(builderOptions =>
                 {
-                    var resourceBuilder = ResourceBuilder.CreateDefault();
-                    configureResource(resourceBuilder);
-                    builderOptions.SetResourceBuilder(resourceBuilder);
-
```

Comment:
Is this for the bug fix? If so, consider adding to the Changelog.

Reply:
It's not a bug, will get this updated under other changes.